### ISSUE TITLE
Gowns Model

### ISSUE THREAD

**Issue by jcasado6** [jserrao/modern-trousseau-gatsby#37]

**Comment by jserrao** [comment]
Gown taxonomies include:

Category (multi-select)
- Seasons (user editable)
- Classics
- Trending (could be dynamic, if you want to code that)

Silhouette (multi-select)
- A-Line
- Sheath
- Ball Gown
- Fit and Flare
- Vintage Style
- Minidress

Details (multi-select)
- Long Sleeves
- Lace
- Illusion
- Back Details
- V-neckline

**Comment by jserrao** [comment]
@mamady310 - think about Accessories and where they fit in. Above are the taxonomies you will need to use.

**Comment by jserrao** [comment]
Conversation around Accessories

TL;DR - Just make Accessories

Full conversation:
John Serrao  3:09 PM
Yeah, that's a good call actually - I found all the taxonomies in the comps.

Karadi  3:10 PM
AlexG +1

John Serrao  3:10 PM
What do you make of these accessories @Karadi? My thought is...haven't seen any content so it's hard to model
new messages

Karadi  3:10 PM
We also have it as part of the gowns and anything else will be a query.

John Serrao  3:12 PM
I guess you're thinking it's a filter because the gown will either have a belt or it wont have a belt?
3:12
But really, it seems like an additional thing - like you would add a belt to a dress?

John Serrao  3:34 PM
@channel - a few more thoughts on accessories. I hate saying this but @AlexG is probably right. I think if we tell the client what we want to do, and push them to have the conversations in store we will have a better build.

AlexG  3:35 PM
I'm always right and the sooner you understand this the sooner we can get back on track

John Serrao  3:36 PM
I think the accessory taxonomy is really a multi select of bools - when you are entered a gown as a contentful user, you would see maybe checkboxes? 'Does this dress have veil options', 'Does this dress have belt options?' and then we could use the single gown model with the accessories dropdown

**Comment by jserrao** [comment]
https://app.contentful.com/spaces/2n4rer1mpqy8/content_types/gowns/fields
